cli/args.go

```diff
@@ -87,6 +87,11 @@ func parseTerragruntOptionsFromArgs(args []string, writer, errWriter io.Writer)
 		return nil, err
 	}
 
+	excludeDir, err := parseStringArg(args, OPT_TERRAGRUNT_EXCLUDE_DIR, os.Getenv("TERRAGRUNT_EXCLUDE_DIR"))
```

Comment:
Should you be able to specify `--terragrunt-exclude-dir` multiple times to capture multiple patterns to exclude?